controllers/grid/settings/sections/form/PKPSectionForm.php

```diff
@@ -115,7 +115,10 @@ public function fetch($request, $template = null, $display = false)
         $assignableUserGroups = UserGroup::query()
             ->withContextIds([$request->getContext()->getId()])
             ->withRoleIds($this->assignableRoles)
-            ->withStageIds([WORKFLOW_STAGE_ID_SUBMISSION])
+            ->withStageIds([
+                // WORKFLOW_STAGE_ID_SUBMISSION for OJS/OMP and WORKFLOW_STAGE_ID_PRODUCTION for OPS, see pkp/pkp-lib#10874
+                min(Application::getApplicationStages())
```

Comment:
I think it would be better not to use mathematical operators on arbitrarily defined constants. If you like, you could use the ordering of the items returned by `getApplicationStages` -- e.g. call `array_shift` on its results to determine the first stage.